src/public/video.ts

```diff
@@ -114,6 +148,15 @@ export namespace video {
   export function registerForVideoEffect(callback: VideoEffectCallBack): void {
     ensureInitialized(FrameContexts.sidePanel);
     registerHandler('video.effectParameterChange', callback);
+    sendMessageToParent('video.registerForVideoEffect');
```

Comment:
Why do you need to send two messages to Teams here? The registerHandler call is already sending a message that you can handle on the Teams side.

Reply:
the registerHandler has called the sendMessageToParent(), we don't need call SendMessageToParent() again. Just change the handler of 'video.registerForVideoEffect' to 'video.effectParameterChange'